pages/History.qml

```diff
@@ -633,7 +633,7 @@ Rectangle {
                                 MoneroComponents.TextPlain {
                                     font.family: MoneroComponents.Style.fontRegular.name
                                     font.pixelSize: 15
-                                    text: (isout ? qsTr("Sent") : qsTr("Received")) + translationManager.emptyString
+                                    text: (isout ? (isFailed ? qsTr("Sent (Failed)") : (isPending ? qsTr("Sent (Pending)") : qsTr("Sent"))) : qsTr("Received")) + translationManager.emptyString
```

Comment:
Breaks the translation. Displaying `Sent (Failed)` looks good. Just refactor this to reuse already translated `Sent`, `Failed` and `Pending` strings.